usecases/backup/coordinator_test.go

```diff
@@ -71,6 +71,8 @@ func TestCoordinatedBackup(t *testing.T) {
 	t.Run("Success", func(t *testing.T) {
 		t.Parallel()
 		fc := newFakeCoordinator(nodeResolver)
+		fc.selector.On("Shards", ctx, classes[0]).Return(nodes, nil)
+		fc.selector.On("Shards", ctx, classes[1]).Return(nodes, nil)
```

Comment:
comment(non-blocking): I don't think there is any place where we assert that all the calls stated in `On` have happened. It would be nice to add a call to https://pkg.go.dev/github.com/stretchr/testify/mock#AssertExpectationsForObjects to ensure we are doing what we expect

Reply:
I don't think these assertion are need, since we are already verifying that the end result is correct